src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -123,6 +126,17 @@ export function ReceiptModal({
               <StatusField order={order} />
             </styledEl.Field>
 
+            {order.receiver && (
+              <styledEl.Field>
+                <FieldLabel label="Reciever" tooltip={tooltips.RECEIVER} />
```

Comment:
Also, we call it `recipient` in the UI
![image](https://github.com/cowprotocol/cowswap/assets/43217/9d3f7b02-e9db-4455-857b-05ba8e0a5cf9)

Better to keep the same name

Reply:
Fixed